# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/RangeUtils.cpp
# Commit: 260574c2a78e
# Full Hash: 260574c2a78e619e6c9815a797a8df7086004994
# Author: Serban Stanca <sstanca@mozilla.com>
# Date: 2025-05-09 04:15:00
# Regressor Bug: 1932150
# File Overlap Count: 1
# Description:
#   Revert "Bug 1932150 - Update ranges to correctly set and update nodes for flattened selection r=jjaschke,smaug,dom-core"
#   
#   This reverts commit 0221ba097eb5884f3efe76c607cb0cc7c9170635.
# ==============================================================================

diff -r d699f1b53a1e -r 260574c2a78e dom/base/RangeUtils.cpp
--- a/dom/base/RangeUtils.cpp	Thu May 08 22:53:05 2025 +0300
+++ b/dom/base/RangeUtils.cpp	Thu May 08 22:53:06 2025 +0300
@@ -10,7 +10,6 @@
 #include "mozilla/dom/AbstractRange.h"
 #include "mozilla/dom/Document.h"
 #include "mozilla/dom/ShadowRoot.h"
-#include "mozilla/dom/HTMLSlotElement.h"
 #include "nsContentUtils.h"
 #include "nsFrameSelection.h"
 